percona-server-8.0/ps-entry.sh

```diff
@@ -227,4 +227,23 @@ if [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
   fi
 fi
 
+
+CALL_HOME_OPTIONAL_PARAMS=
+
+if [ ! -z "$PERCONA_INSTANCE_ID" ]; then
```

Comment:
where this params value is set ?

Reply:
this is the same story like with `-e MYSQL_ROOT_PASSWORD=secret` which can be passed when the container is created like
`docker run --name container-name -e MYSQL_ROOT_PASSWORD=secret -e PERCONA_INSTANCE_ID=<uuid4>`

Reply:
but this value can be set by our operators only, user will set it almost never.

Reply:
Yes, that's the intention.

Reply:
ok, thanks